devito4pytorch/devito_wrapper.py

```diff
@@ -29,8 +33,8 @@ def backward(ctx, grad_output):
         rec.data[:] = grad_output[:]
 
         # Adjoint linearized modeling
-        u0 = ctx.solver.forward(save=True)[1]
-        g = ctx.solver.gradient(rec, u0)[0].data
+        u0 = ctx.solver.forward(save=True, vp=ctx.model.vp)[1]
```

Comment:
why does this one need vp?

Reply:
I was unsure about this. Does not really. Will fix other instances of it as well.